pkg/pulumiyaml/ast/template_test.go

```diff
@@ -118,3 +118,103 @@ func TestComponentParsing(t *testing.T) {
 	require.Len(t, template.Components.Entries[1].Value.Outputs.Entries, 1)
 	require.Equal(t, "bucketEndpoint", template.Components.Entries[1].Value.Outputs.Entries[0].Key.Value)
 }
+
+const componentSchemaExample = `
+name: yaml-plugin
+description: A YAML plugin
+runtime: yaml
+components:
+  aComponent:
+    description: "A component"
+    config:
```

Comment:
Do we have to call it `config` not `inputs`?

Reply:
It's a very simple change.  I mainly called this `config` to keep the familiarity with current YAML program, but I can change it.  Let's keep that to a separate PR though so we can discuss. (None of this can be used right now, so we're free to change it)